pkg/scheduler/framework/v1alpha1/framework_test.go

```diff
@@ -1064,6 +1064,66 @@ func TestRunBindPlugins(t *testing.T) {
 			injects:    []Code{Skip, Success, Error},
 			wantStatus: Success,
 		},
+		{
+			name:       "NoBindPlugin",
+			injects:    []Code{},
+			wantStatus: Success,
+		},
+		{
+			name: "UnshedulableBindPlugin",
+			injects:  []Code{Unschedulable},
+			wantStatus: Error,
+		},
+		{
+			name: "SkipBindPlugin",
+			injects:  []Code{Skip},
+			wantStatus: Skip,
+		},
+		{
+			name: "ErrorBindPlugin",
+			injects:  []Code{Error},
+			wantStatus: Error,
+		},
+		{
+			name: "SkipSuccessBindPlugins",
+			injects: []Code{Skip, Success},
+			wantStatus: Success,
+		},
+		{
+			name: "SkipUnschedulableAndUnresolvedBindPlugin",
+			injects: []Code{Skip, UnschedulableAndUnresolvable},
+			wantStatus: Error,
+		},
+		{
+			name: "SuccessSuccessBindPlugin",
+			injects: []Code{Success, Success},
+			wantStatus: Success,
+		},
+		{
+			name: "ErrorErrorBindPlugin",
+			injects: []Code{Error, Error},
+			wantStatus: Error,
+		},
+		{
+			name: "SuccessErrorBindPlugin",
+			injects: []Code{Success, Error},
+			wantStatus: Success,
+		},
+		{
+			name: "SuccessUnschedulableBindPlugin",
+			injects: []Code{Success, Error},
+			wantStatus: Success,
+		},
+		{
+			name: "ErrorSuccessBindPlugin",
```

Comment:
in general, put in the name what should happen. But also, some tests seem redundant from the ones we already have.

Reply:
Okay. I have made some changes, can you please take another look?